backend/inspirehep/indexer/tasks.py

```diff
@@ -7,7 +7,13 @@
 
 import structlog
 from celery import shared_task
-from elasticsearch import NotFoundError
+from elasticsearch import (
+    ConflictError,
+    ConnectionError,
+    ConnectionTimeout,
+    NotFoundError,
+    RequestError,
```

Comment:
Where are these used? Some of them are overridden by `sqlalchemy` imports.

Reply:
I don't see any of them being overridden by sqlalchemy imports...

3 of them are used here https://github.com/inspirehep/inspirehep/pull/1614/files/0d4430201ab14d3a05185748ce1ced8d909ddb1f#diff-ef7ea812318a6a720d1f514bad6aca84b23b792ec7a70291a22c90d4b84ff85dR42-R44
One here: https://github.com/inspirehep/inspirehep/pull/1614/files/0d4430201ab14d3a05185748ce1ced8d909ddb1f#diff-ef7ea812318a6a720d1f514bad6aca84b23b792ec7a70291a22c90d4b84ff85dR97
And one here: https://github.com/inspirehep/inspirehep/pull/1614/files/0d4430201ab14d3a05185748ce1ced8d909ddb1f#diff-ef7ea812318a6a720d1f514bad6aca84b23b792ec7a70291a22c90d4b84ff85dR92

Reply:
Do I miss something?

```
from elasticsearch import (
     ConflictError,
     ConnectionError,
     ConnectionTimeout,
     NotFoundError,
     RequestError,
)

from sqlalchemy.exc import (
     DisconnectionError,
     UnboundExecutionError,		     
     ResourceClosedError,		   
     OperationalError,		     
     ConnectionError,
     ConnectionTimeout,
     RequestError,
 )		
```

Both imports have `RequestError`, `ConnectionTimeout`, `ConnectionError`

Reply:
It would be good to re-enable flake8. I realized when doing the `pyproject.toml` switch that we don't automatically run it.

Reply:
indeed, we were running `flake8` on unit tests but it seems that we didn't migrate it to the github actions

Reply:
> Do I miss something?
> 
> ```python
> from elasticsearch import (
>      ConflictError,
>      ConnectionError,
>      ConnectionTimeout,
>      NotFoundError,
>      RequestError,
> )
> 
> from sqlalchemy.exc import (
>      DisconnectionError,
>      UnboundExecutionError,		     
>      ResourceClosedError,		   
>      OperationalError,		     
>      ConnectionError,
>      ConnectionTimeout,
>      RequestError,
>  )		
> ```
> 
> Both imports have `RequestError`, `ConnectionTimeout`, `ConnectionError`

Am I that tired? I don't see those imports from sqlalchemy in this file...